adagrams/game.py

```diff
@@ -1,11 +1,89 @@
+from random import choices
+
+LETTER_POOL = {
+    "A": 9,
+    "B": 2,
+    "C": 2,
+    "D": 4,
+    "E": 12,
+    "F": 2,
+    "G": 3,
+    "H": 2,
+    "I": 9,
+    "J": 1,
+    "K": 1,
+    "L": 4,
+    "M": 2,
+    "N": 6,
+    "O": 8,
+    "P": 2,
+    "Q": 1,
+    "R": 6,
+    "S": 4,
+    "T": 6,
+    "U": 4,
+    "V": 2,
+    "W": 2,
+    "X": 1,
+    "Y": 2,
+    "Z": 1,
+}
+ONLY_LETTERS = [letter for letter in LETTER_POOL.keys()]
+ONLY_WEIGHTS = [num for num in LETTER_POOL.values()]
+LETTER_VALUE = {
+    1: ["A", "E", "I", "O", "U", "L", "N", "R", "S", "T"],
```

Comment:
[idea] You could get away with

```suggestion
    1: "AEIOULNRST",
```

Strings support the `in` operator. 😉